source/slang/slang-check-overload.cpp

```
@@ -13,6 +13,30 @@
 namespace Slang
 {
 
+// True for BFloat16/FloatE4M3/FloatE5M2 scalars or vectors of them.
+static bool _isStorageOnlyFloatType(Type* type)
+{
+    if (!type)
```

Cppcheck (2.20.0)

[error] There is an unknown macro here somewhere. Configuration is required. If FIDDLE is a macro then please configure it.

(unknownMacro)